pool-interface/src/lib.rs

```diff
@@ -34,18 +34,21 @@ pub enum Error {
     CollateralNotCoverNewBorrow = 14,
     BadPosition = 15,
 
-    InvalidReserveParams = 16,
-    ReserveLiquidityNotZero = 17,
-
-    ReservesMaxCapacityExceeded = 18,
-    NoPriceForAsset = 19,
+    ReserveLiquidityNotZero = 16,
+    ReservesMaxCapacityExceeded = 17,
+    NoPriceForAsset = 18,
 
     MathOverflowError = 100,
     PriceMathOverflow = 101,
     ValidateBorrowMathError = 102,
     CalcAccountDataMathError = 103,
     CalcInterestRateMathError = 104,
     AssetPriceMathError = 105,
+
+    MustBeLte10000Bps = 106,
```

Comment:
As discussed internally: it's better to avoid using values in Error names. As this value refers to `PERCENTAGE_FACTOR` const we need public readonly function for getting this const to make contract interaction more descriptive

Reply:
Also let's split errors into blocks, with 100 in each with the appropriate prefixes